### DISCUSSION TITLE
Multiple tags per card, questions & docs on the tag system

### DISCUSSION THREAD

**Discussion by janek** [st3v3nmw/obsidian-spaced-repetition#363] [Q&A] [answered]
hi! Thank you very much for your work on this. 

I'm having trouble organising my cards. I realised they can be nested, which is already great, but one feature wish I'd have is to have multiple tags per card. As in, I could have #phys and #maths, but also #trivia, and #trivia applies to some cards from  #phys and some from #math. I could then learn #trivia about anything.

I came from an app called NeuraCache and I think that's probably why I immediately thought about it. I found their tag system very straightforward and useful (see screens). It did require writing tags out for every card though. I definitely see the advantage of not doing that, but it would be great to have both (a note with tags that apply to all cards on it, but also additional tags per card).

Also, if working with single (nested) tags only, can I have #flashcards/math/serious and #flashcards/math/trivia in one note? Or should I create a separate note for every tag? (just making sure)

In general, of course, it would be great to have a "Organising cards with tags or folders" section in the wiki. For me it was a point of friction and I only found out about nesting tags from a discussion thread (that was about an unrelated topic).

| Neuracache: syntax incl. tags  | Tags view in app |
| ------------- | ------------- |
| <img width="1000" alt="image" src="https://user-images.githubusercontent.com/2146927/147871287-e9efe666-28dc-4af5-a046-93719ac9219a.png">  | <img width="400" alt="image" src="https://user-images.githubusercontent.com/2146927/147871293-18576f02-d5c2-4c6d-8dde-6f16a63162fa.PNG"> 

____

**Comment by Eirikfesker** [comment]
Hello, i need this feature too.

e.g. im psychology student and i have some cards/questions that can appear in other module-tests too.
when i tah my notes with #flashcards/psy1 and #flashcards/psy2 only the first one appears in the list. what can i do?

**Comment by Celeblith** [comment]
Just spent an hour organizing my flashcards on the assumption that I'd be able to use multiple tags per deck. Turns out I was sorely mistaken. Please, please, please add this feature.

**Comment by ronzulu** [comment, marked as answer]
Hi all

This has been implemented and is available for beta testing at:
[Support multi-deck cards #495, and multiple decks in a single note #705](https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/834) 

Cheers
Ronny
